<launch>
  <arg name="path_to_gt_map" default="/home/kirill/TopoSLAM/GT/2n8kARJN3HM/map_cropped_0_600_300_900.png"/>
  <!-- arg name="path_to_gt_map" default="/home/kirill/TopoSLAM/GT/2n8kARJN3HM/map_cropped_0_600_300_900.png"/ -->
  <!-- arg name="path_to_gt_map" default="/home/kirill/TopoSLAM/mipt_map_gt_1800_4000_1800_2300.png"/ -->
  <!-- arg name="path_to_save_json" default="/home/kirill/TopoSLAM/toposlam_ws/src/simple_toposlam_model/graph_jsons_new/mipt2"/ -->
  <arg name="path_to_save_json" default="/home/kirill/TopoSLAM/toposlam_ws/src/simple_toposlam_model/graph_jsons_new/2n8kARJN3HM"/>
  <arg name="config_file" default="habitat_mp3d.yaml"/>

  <node pkg="prism_topomap" type="prism_topomap_node.py" name="prism_topomap_node" output="screen">
    <param name="path_to_gt_map" value="$(arg path_to_gt_map)"/>
    <param name="path_to_save_json" value="$(arg path_to_save_json)"/>
    <param name="config_file" value="$(arg config_file)"/>
  </node>

  <node pkg="prism_topomap" type="navigation.py" name="navigation_server" output="screen">
    <param name="rate" value="2"/>
    <param name="tolerance" value="0.5"/>
    <param name="timeout" value="1000"/>
  </node>

  <!-- node pkg="theta_star_planner" type="theta_star_planner_node" name="local_path_planner" output="screen">
    <param name="map_topic" value="local_grid"/>
    <param name="task_topic" value="local_planning_task"/>
    <param name="path_topic" value="local_path"/>
    <param name="map_frame" value="last_vertex"/>
  </node -->
</launch>